src/v2/schema/application.js

```diff
@@ -88,7 +84,6 @@ export const resolver = {
     labels: (parent) => (parent['app.label'] ? parent['app.label'].split(';').map((l) => l.trim()) : []),
     name: (parent) => parent['app.name'],
     namespace: (parent) => parent['app.namespace'],
-    selfLink: (parent) => parent['app.selfLink'],
```

Comment:
Let's use this as a temporary step. It's far from the ideal deprecation process, but better than unexpectedly removing the field. 
```js
selfLink: (parent) => 'selfLink has been removed in K8s 1.20',
```